charts/oauth/templates/configmap.yaml

```
@@ -34,6 +34,9 @@ data:
     frontend:
       dir: /srv/dex/web
       logoURL: theme/logo.svg
+{{- if .Values.dex.frontend }}{{- if .Values.dex.frontend.issuer }}
+      issuer: {{ .Values.dex.frontend.issuer }}
+{{- end }}{{- end }}
```

This will result in a null pointer exception if `dex.frontend` doesn't exist. 
```suggestion
issuer: {{ ((.Values.dex).frontend).issuer | default "dex" }}
```

---

Huh, strange. I specifically wrote the outer if-check (the one on `dex.frontend` only) to avoid that issue. It worked during my testing, if there's no `dex.frontend` the first if-condition will evaluate to false and just not render anything.

But yeah, your solution is much simpler and elegant. I'll change it. :+1: